composer.json

```diff
@@ -13,5 +13,8 @@
     },
     "config": {
         "optimize-autoloader": true
+    },
+    "require": {
+        "ext-curl": "*"
```

Comment:
i think this is already required in magento, but i feel like its not a bad thing to have since our extension depends on it